### ISSUE TITLE
Generate Assessment Plan from SSP

### ISSUE THREAD

**Issue by meganwolf0** [defenseunicorns-labs/lula1#811]
### Is your feature request related to a problem? Please describe.

According to understood OSCAL model process flows, the SSP is the precursor model to the Assessment Plan. This issue is aligned with determining how to generate an Assessment Plan from the SSP. This _potentially_ includes executing Lula Validations to determine contents of the Assessment Plan (and downstream Assessment Results?)

### Describe the solution you'd like

- **Given** an SSP
- **When** `lula generate -f <ssp.yaml> <flags>`
- **Then** an Assessment Plan is created

### Describe alternatives you've considered

There may be another OSCAL model flow, this is possibly unclear? This issue conflicts with #518 where an Assessment Plan is documented as an output of a Component Definition. Which is true? Which do we want to go with if _both_ could be true?

### Additional context

Outstanding Questions - do we need to run `lula validate` under the hood as part of this process? Relates to #518, #514.

Additional point of discussion - How many commands do we want to have/users need to memorize to create OSCAL docs? (should we create an `assess` command or keep as `generate`?)

### Expected Deliverable

- [ ] Discussion and fully scoped issue set on the details of the implementation

**Comment by brandtkeller** [comment]
Likely shares an overlap with #514 and #518.

**Comment by meganwolf0** [comment]
> Likely shares an overlap with https://github.com/defenseunicorns/lula/issues/514 and https://github.com/defenseunicorns/lula/issues/518.

Agree, called out #518 in the description but #514 is probably there as well - seems like the deliverable on this would either scope or close those issues as they relate to the specific `assess` command (https://github.com/defenseunicorns/lula/issues/514 - do we even want to create another command?) and the inputs as a `component-definition` (https://github.com/defenseunicorns/lula/issues/518 - _should_ that be the input or should it be an SSP?)
